### PR TITLE
Identity | Sign in Gate | Copy test and switch

### PR DESCRIPTION
**Pull request by rebecca-thompson**
## What does this change?
Set up the AB Test definition and switch for the Sign In Gate Copy Optimisation test. The actual tests and designs have only been implemented on [DCR](https://github.com/guardian/dotcom-rendering/pull/2622), but the audience for both platforms has to be identical to get relevant results.

### Tested

- [ ] Locally
- [x] On CODE (optional)

### CODE REVIEW

**Comment by rebecca-thompson** [comment]
[Copy Test on the Sign In Gate (M/L)](https://trello.com/c/G15Alj3r/2822-copy-test-on-the-sign-in-gate-m-l)

File: static/src/javascripts/projects/common/modules/experiments/tests/sign-in-gate-copy-opt.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+export const signInGateCopyOpt = {
+    id: 'SignInGateCopyOpt',
+    start: '2021-02-24',
+    expiry: '2021-12-01',
+    author: 'rebecca-thompson',
+    description:
+        'Compare 6 different gate copy changes, on 3rd page view of the day, on simple article templates, with higher priority over banners and epi',
+    audience: 0.2,
```
Comment (alexduf) [inline static/src/javascripts/projects/common/modules/experiments/tests/sign-in-gate-copy-opt.js:8]: Shouldn't this be 0.3? Or is there something I'm missing?

Comment (rebecca-thompson) [reply static/src/javascripts/projects/common/modules/experiments/tests/sign-in-gate-copy-opt.js:8]: the last 10% is for the control (no gate) with a 0.9 offset and 0.1 audience

**Review by alexduf — APPROVED** [review]

**Review by coldlink — APPROVED** [review]
LGTM

Not sure why the build is failing though?

**Pull request merged**